### DISCUSSION TITLE
Add feature to uncheck all tasks

### DISCUSSION THREAD

**Discussion by cdauth** [outline/outline#7715] [Ideas]
I am using an Outline page as a checklist what to pack for a trip. The page contains a task list, and for each item that I pack, I check the corresponding task, which gives me an easy overview over what I still need to pack.

When I go on a new trip, I need to uncheck all tasks before starting to pack again. As I have more than 100 items in the task list, it would be great to have a way to uncheck all tasks at once. This could for example be a button in an inline dialog that opens when clicking the “x of y tasks” byline item on top of the page.

**Comment by tommoor** [comment]
`cmd`+`enter` is the shortcut to toggle a single checkbox, I think we could extend this to toggle N checkboxes that you have within selected state 🤔

**Reply by cdauth** [reply]
Not a bad suggestion!

On a side note, I frequently get confused about that shortcut, since it's the same as for “Done editing”.

**Reply by AverageMarcus** [reply]
I really like this as a suggestion. I wasn't aware of `cmd` + `enter` which helps speed things up somewhat but if we could use the same after highlighting multiple checkboxes that would be awesome. Could this be moved to a feature request? 🙏

**Reply by tommoor** [reply]
Putting Claude on it

**Comment by StefanRiedAV** [comment]
you could put your checklist into a template and create for each trip and new page based on the template. Then you have a fresh unchecked tasklist.
